packages/driver/src/cy/assertions.coffee

```diff
@@ -56,6 +58,35 @@ parseValueActualAndExpected = (value, actual, expected) ->
 
   obj
 
+prepareObjsForDiff = (err) ->
+  if _.isString(err.actual) || _.isString(err.expected)
+    return err
+  ret = {}
+  ret.actual = mochaUtils.stringify(err.actual)
+  ret.expected = mochaUtils.stringify(err.expected)
+  return ret
+
+objToString = Object.prototype.toString
```

Comment:
is there a difference in `Object.prototype.toString` and `mochaUtils.stringify`?

Reply:
added comments to clarify